pandas/tests/frame/methods/test_shift.py

```diff
@@ -524,9 +524,9 @@ def test_datetime_frame_shift_with_freq(self, datetime_frame, frame_or_series):
     def test_period_index_frame_shift_with_freq_error(self, frame_or_series):
         ps = tm.makePeriodFrame()
         ps = tm.get_obj(ps, frame_or_series)
-        msg = "Given freq M does not match PeriodIndex freq B"
+        msg = "Given freq ME does not match PeriodIndex freq B"
         with pytest.raises(ValueError, match=msg):
-            ps.shift(freq="M")
+            ps.shift(freq="ME")
```

Comment:
if it's a periodindex, we should be using `'M'` anyway - can we leave this test as it is?